ckcp/openshift_dev_setup.sh

```diff
@@ -273,38 +268,20 @@ patches:
   echo "OK"
 }
 
-install_pipelines_crds() {
-  install_app pipelines-crds
-}
-
-install_pipelines_controller() {
-  # Create kcp-kubeconfig secret for pipelines controller
-  echo -n "  - Register KCP secret to host cluster: "
-  kubectl create namespace pipelines --dry-run=client -o yaml | kubectl apply -f - --wait &>/dev/null
-  kubectl create secret generic kcp-kubeconfig -n pipelines --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | kubectl apply -f - --wait &>/dev/null 2>&1
-  echo "OK"
-
-  install_app pipelines-controller
-}
-
-install_triggers_crds() {
-  install_app triggers-crds
-}
-
-install_triggers_interceptors() {
-  # Create kcp-kubeconfig secrets for event listener and interceptors so that they can talk to KCP
-  kubectl create secret generic kcp-kubeconfig --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | KUBECONFIG="$KUBECONFIG_KCP" kubectl apply -f - --wait &>/dev/null
-  kubectl create secret generic kcp-kubeconfig -n tekton-pipelines --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | KUBECONFIG="$KUBECONFIG_KCP" kubectl apply -f - --wait &>/dev/null
-
-  install_app triggers-interceptors
-}
+install_openshfit_pipeline() {
+  APP="argocd"
 
-install_triggers_controller() {
-  # Create kcp-kubeconfig secret for triggers controller
-  kubectl create namespace triggers -o yaml --dry-run=client | kubectl apply -f - --wait &>/dev/null
-  kubectl create secret generic kcp-kubeconfig -n triggers --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | kubectl apply -f - --wait &>/dev/null
+  echo -n "  - openshift-pipeline application: "
+  kubectl apply -f "$GITOPS_DIR/$APP/$APP.yaml" --wait >/dev/null 2>&1
+  argocd app wait pipelines-service tektoncd >/dev/null 2>&1
```

Comment:
if we are requesting to install argocd plugin on the user machine only for this check this seems over-killed.

Reply:
@fgiloux the check is not over-killed. As it install argocd plugin by calling the umbrella app `gitops/argocd/argocd.yaml`. It will deploy ArgoCD applications `pipelines-service` and `tektoncd` in sequence. it has to check the both applications. If it just check `tektoncd`, the script will be broken.

Reply:
I did not mean the check. I meant requiring a plugin to be installed for a couple of API calls. But yes the argocd plugin is also used for kcp registration, which is a different story.

Reply:
ah, I got it. thank you for your explanation.